ciao-cli/instance.go

```diff
@@ -506,6 +506,10 @@ func dumpInstance(server *compute.ServerDetails) {
 		fmt.Printf("\tSSH IP: %s\n", server.SSHIP)
 		fmt.Printf("\tSSH Port: %d\n", server.SSHPort)
 	}
+
+	for _, vol := range server.OsExtendedVolumesVolumesAttached {
```

Comment:
We also need to update the instanceTemplateDesc constant.  Otherwise users who type ciao-cli instance list --help will not see that the new field, OsExtendedVolumesVolumesAttached, is available to templates.